packages/playwright-ct-solid/registerSource.mjs

```diff
@@ -54,7 +54,10 @@ function render(component) {
   if (component.kind !== 'jsx')
     throw new Error('Object mount notation is not supported');
 
-  return createComponent(componentFunc, component.props);
+  return createComponent(componentFunc, {
+    children: component.children,
+    ...component.props
```

Comment:
nit: trim children from here?

Reply:
Can I fix this in the next PR? I'm working on the multi-root children feature which touches this part

Reply:
Sadly they always require rebaselines